docs/reference/config-options.md

```diff
@@ -319,3 +319,30 @@ When a `null` value is encountered, it is assumed to be
 `defaultForNull`.
 
 (Introduced in https://github.com/wireapp/wire-server/pull/1811.)
+
+### Locale
+
+
+#### setDefaultLocale (deprecated / ignored)
+
+The brig server config option `setDefaultLocale` has been replaced by `setDefaultUserLocale` and `setDefaultTemplateLocale`. Both settings are optional and `setDefaultTemplateLocale` defaults to `EN` and `setDefaultLocale` defaults to `setDefaultTemplateLocale`. If `setDefaultLocale` was not set or set to `EN` before this change, nothing needs to be done. If `setDefaultLocale` was set to any other language other than `EN` the name of the setting should be changed to `setDefaultTemplateLocale`. 
+
+#### `setDefaultTemplateLocale`
```

Comment:
Which templates are actually complete in wire-server other than EN? i.e. this settings can't be set to anything other than EN at this point in time unless more templates are provided inside wire-server's brig/deb/templates folder, right? Perhaps the description here should make that clearer.